fixes.py

```diff
@@ -302,3 +302,29 @@ def authoritative_fix(self):
         except RuntimeError:
             self.item_report['authoritative_result'] = f'User does not have privileges to change content status. Please use an AGOL account that is assigned the Administrator role.'
             return
+
+    def visibility_fix(self):
+        '''
+        Access item's manager object and set it's defaultVisibility property
+        to True
+
+        Updates item_report with results for this fix:
+        {visibility_result: result}
+        '''
+
+        if self.item_report['visibility_fix'].casefold() != 'y':
```

Comment:
**question**: is there a reason to use strings here instead of a boolean? These are binary options right? If so boolean would be more appropriate.

Reply:
It uses a nested dictionary with these smaller dictionaries to both pass the fixes to the `fixer` object and also to write the report. I initially figured 'Y' was more report-file-friendly than 'True' or 'False', so I've stuck with that convention through the whole project now.